recipes/hipace/meta.yaml

```diff
@@ -0,0 +1,93 @@
+{% set name = "hipace" %}
+{% set version = "23.03" %}
+{% set build = 0 %}
+{% set sha256 = "f32104af1e7ca33b7ef8a3fd7964ee62db13a68a00af305d4ace5d0e86e2efb2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Hi-PACE/hipace/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: {{ build }}
+  run_exports:
+    # releases are not (yet) compatible
+    - hipace {{ version }}
```

Comment:
```suggestion
    - {{ pin_compatible('hipace', max_pin='x.x.x') }}
```

Reply:
Oh, this [complains below](https://github.com/conda-forge/staged-recipes/pull/22171#issuecomment-1541176100):
```
    pin_subpackage should be used instead of pin_compatible for hipace because it is one of the known outputs of this recipe: ['hipace'].
```

Will try `pin_subpackage`

Reply:
Yes. Listen to the linter. 😄